src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -75,7 +75,7 @@ internal Symbol GetWellKnownTypeMember(WellKnownMember member)
             return _lazyWellKnownTypeMembers[(int)member];
         }
 
-        internal NamedTypeSymbol GetWellKnownType(WellKnownType type)
+        internal NamedTypeSymbol GetWellKnownType(WellKnownType type, bool ignoreCorLibraryDuplicatedTypes = false)
```

Comment:
Aleksey pointed out that we are in a Compilation here, so no need to thread this through. We can just grab this option from the compilation.